src/core/modules/Constellation.cpp

```diff
@@ -474,5 +613,237 @@ StelObjectP Constellation::getBrightestStarInConstellation(void) const
 			maxMag = Mag;
 		}
 	}
+	for (int i=0; i<hullExtension.size();++i)
```

Comment:
I get the following warning here:
```
src/core/modules/Constellation.cpp:616:24: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<QSharedPointerNoDelete<StelObject> >::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
  616 |         for (int i=0; i<hullExtension.size();++i)
      |                       ~^~~~~~~~~~~~~~~~~~~~~
```
So either `i` should be `unsigned`, or the `.size()` should be converted to `int`.